client/my-sites/site-settings/form-general.jsx

```diff
@@ -151,7 +152,8 @@ const FormGeneral = React.createClass( {
 							valueLink={ this.linkState( 'blogdescription' ) }
 							disabled={ this.state.fetchingSettings }
 							onClick={ this.onRecordEvent( 'Clicked Site Site Tagline Field' ) }
-							onKeyPress={ this.onRecordEventOnce( 'typedTagline', 'Typed in Site Site Tagline Field' ) } />
+							onKeyPress={ this.onRecordEventOnce( 'typedTagline', 'Typed in Site Site Tagline Field' ) }
```

Comment:
I tracked this down to a change dated 2015/02/06. :) (pre-oss/3735) You're free to ping Bob if you're interested in knowing whether this is still used.